src/main/java/com/codessquad/qna/controller/PostController.java

```diff
@@ -2,36 +2,44 @@
 
 import com.codessquad.qna.dto.PostDto;
 import com.codessquad.qna.entity.Post;
+import com.codessquad.qna.entity.User;
 import com.codessquad.qna.exception.CanNotFindPostException;
 import com.codessquad.qna.service.PostService;
+import com.codessquad.qna.util.HttpSessionUtils;
 import com.codessquad.qna.util.Mapper;
-import com.sun.org.apache.xpath.internal.operations.Mod;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.Optional;
+import javax.servlet.http.HttpSession;
 
 @Controller
+@RequestMapping("/questions")
 public class PostController {
 
-    private Logger logger = LoggerFactory.getLogger(PostController.class);
+    private static final Logger logger = LoggerFactory.getLogger(PostController.class);
+    private final PostService postService;
 
-    @Autowired
-    PostService postService;
+    public PostController(PostService postService) {
+        this.postService = postService;
+    }
+
+    @GetMapping("/form")
+    public String getQnaForm() {
+        return "qna/form";
+    }
 
     /**
      * 질문 게시글을 게시판에 등록합니다.
      * @param postDto
      * @return
      */
-    @PostMapping("/questions")
-    public String addPost(@ModelAttribute PostDto postDto) {
-        Post post = Mapper.mapToPost(postDto);
-        postService.addPost(post);
+    @PostMapping("")
```

Comment:
```suggestion
    @PostMapping
```
요렇게도 쓸 수 있는데, 저는 Roach가 사용한 방식을 선호하긴 해요 ㅎㅎ